app/public/css/styles.css

```diff
@@ -0,0 +1,70 @@
+body {
+  font-family: 'Raleway', sans-serif;
+  background: white;
+}
+
+header {
+  background: black;
+  color: white;
+}
+
+header h1 {
+  font-size: 75px;
+  float: right;
+  display: inline-block;
+  padding: 20;
+}
+
+header .img {
+  float: right;
+  display: inline-block;
+  margin: auto;
+  height: 10px;
+  width: 10px;
+}
+
+#filter-or-add {
+  padding: 10px;
+  display: flex;
+  justify-content: space-around;
+  background: black;
+  color: white;
+}
+
+#statistics {
+  display: flex;
+  flex-direction: column;
+}
+
+#statistics h1 {
+  display: flex;
+  justify-content: center;
+}
+
+.three-stats {
+  margin: 10px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: flex-start;
+}
+
+.hometowns-text {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+#profiles {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+}
+
+#profiles .card {
+  border: solid;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+}
```

Comment:
generally speaking, very good job with CSS. in Week 4 we'll take a peek at "Sass", aka SCSS, which will let you embed nested attributes which then get compiled into proper css, so you can do neat things like:

```scss
header {
  background: black;
  color: white;

  h1 {
    font-size: 75px;
    float: right;
    display: inline-block;
    padding: 20;
  }

  .img {
    float: right;
    display: inline-block;
    margin: auto;
    height: 10px;
    width: 10px;
  }
}
```

Reply:
That SCSS would turn into the CSS you've already built, but makes it easier to organize etc if you do nesting. SCSS can also do neat things with variables so you can declare color schemes in variable names like `$my_shade_of_blue` and then use that in your CSS instead of copying `#0099cc` everywhere